src/checks/lowercase_key.rs

```diff
@@ -8,7 +8,7 @@ pub(crate) struct LowercaseKeyChecker {
 impl Default for LowercaseKeyChecker {
     fn default() -> Self {
         Self {
-            template: String::from("The {} key should be in uppercase"),
+            template: String::from("LowercaseKey: The {} key should be in uppercase"),
```

Comment:
In the future I would like to have the ability to disable some checks by their names.
To do that, all checks should have the `name` field.
Please, move the check's name from the template to the special field.
Also, I would like to have a method to show the check's name in the message, for example:
```rust
impl LowercaseKeyChecker {
    fn message(&self, key: &str) -> String {
      // code...
    }
}
```

Reply:
Nice, i'll fix it